cves/CVE-2011-3923.yml

```diff
@@ -182,8 +177,11 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      Since the HTTP paramaters were not being properly escaped, OLGN functiona calls
+      able to be made. If the inputs were being validated/sanitized, this would've been
+      avoided entierly.
```

Comment:
Small spelling error on "entirely"

Reply:
Thanks! I will fix this in the next commit.